integration-test/run_integration_test.sh

```diff
@@ -1,10 +1,10 @@
 # run from this directory
-# use -local for run docker-compose-local.yml
-# example: sh ./run_integration_test.sh -local
-cd ..
+# example: sh ./run_integration_test.sh <local|dev|uat|prod>
+
 
 # create containers
-docker-compose -f ./docker-compose$1.yml up -d --remove-orphans --force-recreate
+cd ../docker || exit
```

Comment:
could be better something like
```
if [[ "$(pwd)" =~ .*"docker".* ]]; then
    cd ..
fi

```
in order to remove the constraint of folder positioning?